doc/users/explain/interactive.rst

```diff
@@ -227,6 +227,29 @@ Preserve aspect ratio              hold **CONTROL** when panning/zooming with mo
 ================================== ===============================
 
 
+Position Format
+---------------
+
+The location of the cursor is show in the UI and generated via the
+`~axes.Axes.format_coord` method which in turn calls the
+`~axes.Axes.format_xdata` and `~axes.Axes.format_ydata` methods.  The hard
+coded format in `~axes.Axes.format_coord` is ``f'x={formtated_x}
+y={formatted_y}'``.
+
+To easily customize how the x and y values are formatted, you can set the
+`.axes.Axes.fmt_xdata` and `.axes.Axes.fmt_ydata` attributes on the
+`~axes.Axes` instance.  The values are expected to be functions that
+take a float and return a string.  For example ::
+
+  fig, ax = plt.subplots()
+  ax.set_ylim(-5, 5)
+  ax.fmt_ydata = lambda v: f'{v:.3g}' if v > 0 else f'({abs(v):.3g})'
```

Comment:
```suggestion
  ax.fmt_ydata = lambda v: f'{v:.3g}' if v >= 0 else f'({abs(v):.3g})'
```
?

(Not really relevant, but as I happened to see it... And it will consistent with the text below.)

Reply:
I think perhaps this should not use `abs`, as it confused me why you didn't just use it directly instead of having the `if`, since the parentheses weren't initially obvious. But using `-v` might make it clear that you're doing the `if` manually on purpose.

Reply:
I think my point was missed(?). It doesn't, strictly, make sense to negate a zero value. Not that it matters (although there is -0.0 in floating-point), but just to make the example a little bit more "logical".